src/go/rpk/pkg/config/rpk_yaml.go

```diff
@@ -145,9 +145,10 @@ type (
 	}
 
 	RpkCloudCluster struct {
-		Namespace string `json:"namespace" yaml:"namespace"`
-		Cluster   string `json:"cluster" yaml:"cluster"`
-		Auth      string `json:"auth" yaml:"auth"`
+		ID         string `json:"id" yaml:"id"`
+		Name       string `json:"name" yaml:"name"`
+		Type       string `json:"type" yaml:"type"`
+		ClusterURL string `json:"cluster_url,omitempty" yaml:"cluster_url,omitempty"`
```

Comment:
Does rpk already have infrastructure to dot he right thing on version mismatches? I assume yes because this is already >v1?

Reply:
Yeah rpk will fail to start if the rpk version is different in the yaml